cluster/saltbase/salt/kube-proxy/initd

```diff
@@ -40,6 +40,13 @@ DAEMON_USER=root
 #
 do_start()
 {
+        # Avoid a potential race at boot time when both monit and init.d start
+        # the same service
+        PIDS=$(pidof $DAEMON)
+        for PID in "${PIDS}"; do
```

Comment:
Oops, same mistake again. Fixed. Thanks for catching this, and luckily we only have one for each daemon here. :-(